### PR TITLE
[Minor] Add miing parts to license

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Update license to correctly list Vben.

### Why are the changes needed?

Add needed information to files and LICENSEs

Fix: # N/A

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Compiled locally.

### CODE REVIEW

File: web/lib/utils/index.js [hunk 1]
```diff
@@ -64,6 +64,10 @@ export const genUpdates = (originalData, newData) => {
   return updates
 }
 
+/*
+ * The following functions originally come from the MIT licensed Vben project.
+ */
+
 export const hasNull = obj => {
```
Comment (ch3yne) [inline web/lib/utils/index.js:71]: The `hasNull` function not copied from anywhere, copied functions are `deepMerge`, `setObjToUrlParams`.

Comment (justinmclean) [reply web/lib/utils/index.js:71]: The tool I'm using suggests it was, perhaps it was copied from somewhere else?

Comment (justinmclean) [reply web/lib/utils/index.js:71]: I unable to find where it might come from, but it it doe come from StackOverlow or anywhere else we need to know.

Comment (ch3yne) [reply web/lib/utils/index.js]: This function is quite simple and universal.
I don't understand why there would be any copyright ownership involved.

**Review by ch3yne — APPROVED** [review]
LGTM - Thanks for the help!

File: web/lib/utils/index.js (hunk 1 above)
Comment (justinmclean) [reply web/lib/utils/index.js]: It doesn't matter how simple you think it is, copyright is still likely to apply.

Comment (justinmclean) [reply web/lib/utils/index.js]: And if it was copied from somewhere whose license is not compatible with the Apache license, then we also have a problem.

Comment (ch3yne) [reply web/lib/utils/index.js]: This function wasn't copied from anywhere.

**Pull request merged**